doc/topics/topology/syndic.rst

```diff
@@ -4,31 +4,37 @@
 Salt Syndic
 ===========
 
+.. warning::
+
+    Syndics are supported when running ``salt`` as the root/service user on the
+    Master of Masters, but :ref:`external auth or publisher_acl<acl-eauth>`
+    may be unstable or not fully supported.
+
+    Ensure minions have unique names under different Syndics. Using the same
+    name for minions under different Syndics is not supported as currently
+    documented. For example, if ``syndic_a`` and ``syndic_b`` both have a
+    ``minion_1``, this behavior is undocumented and unsupported.
+
+    Any other undocumented Syndic implementation should be considered
+    unsupported, such as using a Syndic as a Master of Masters.
+
 The most basic or typical Salt topology consists of a single Master node
 controlling a group of Minion nodes.  An intermediate node type, called Syndic,
 when used offers greater structural flexibility and scalability in the
 construction of Salt topologies than topologies constructed only out of Master
 and Minion node types.
 
-A Syndic node can be thought of as a special passthrough Minion node.  A Syndic
-node consists of a ``salt-syndic`` daemon and a ``salt-master`` daemon running
-on the same system.  The ``salt-master`` daemon running on the Syndic node
-controls a group of lower level Minion nodes and the ``salt-syndic`` daemon
-connects higher level Master node, sometimes called a Master of Masters.
+A Syndic node is a special passthrough Minion node.  A Syndic node consists of
+a ``salt-syndic`` daemon and a ``salt-master`` daemon running on the same
+system.  The ``salt-master`` daemon running on the Syndic node controls a group
+of lower level Minion nodes and the ``salt-syndic`` daemon connects higher
```

Comment:
the ``salt-syndic`` daemon connects to a higher

Reply:
Gotta love long lived typos. Fixed :+1: